src/content/docs/kubernetes-pixie/auto-telemetry-pixie/advanced-configuration/manage-pixie-data.mdx

```diff
@@ -86,7 +86,7 @@ Learn more about the available parameters for the `newrelic-pixie` Helm chart [h
 
 When you deploy Auto-telemetry with Pixie, you're actually enabling the Auto-telemetry with Pixie Helm chart, as well as other New Relic components included in the New Relic infrastructure bundle. The [Pixie Edge Module (PEM)](https://docs.pixielabs.ai/tutorials/custom-data/distributed-bpftrace-deployment#background) pods are deployed to the cluster as a Kubernetes DaemonSet. This means that by default, a pod is scheduled to every cluster node and is responsible for collecting all of the observability metrics for that node.
 
-In Kubernetes, you can assign pods to a specific subset of cluster nodes using nodeSelectors, taints/tolerations, and node affinity/anti-affinity. That way, you'll only collect metrics for the nodes you choose, instead of every node. This is helpful if you only want to deploy Auto-telemetry with Pixie to five of your ten cluster nodes, for example. Maybe the designated five nodes are responsible for high-priority workloads, or perhaps you're running both Linux and Windows nodes in your cluster and only want to deploy on the Linux nodes, since Windows nodes are not currently supported. You can now assign pods to a subset of nodes by providing an additional option to the guided installation command. This option passes an escaped JSON string to the Auto-telemetry with Pixie chart, which enables a `nodeSelector` that only schedules the PEM DaemonSet on nodes with the label `pixie=allowed`.
+In Kubernetes, you can assign pods to a specific subset of cluster nodes using nodeSelectors, taints/tolerations, and node affinity/anti-affinity. That way, you'll only collect metrics for the nodes you choose, instead of every node. This is helpful if you only want to deploy Auto-telemetry with Pixie to five of your ten cluster nodes, for example. Maybe the designated five nodes are responsible for high-priority workloads, or perhaps you're running both Linux and Windows nodes in your cluster and only want to deploy on the Linux nodes, since Windows nodes are not currently supported. You can now assign pods to a subset of nodes by providing an additional option to the guided install command. This option passes an escaped JSON string to the Auto-telemetry with Pixie chart, which enables a `nodeSelector` that only schedules the PEM DaemonSet on nodes with the label `pixie=allowed`.
```

Comment:
I'd consider breaking up this paragraph. It's quite dense and breaking it up could make it easier to parse.

Reply:
Great, thank you @akristen!